src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.model;
+
+import java.util.NoSuchElementException;
+
+public class FruitTransaction {
+    private static final String ILLEGAL_ARGUMENT_EXCEPTION_MESSAGE = "Quantity can't be negative";
+    private static final String NO_SUCH_ELEMENT_EXCEPTION_MESSAGE
```

Comment:
1) This should not be stored inside FruitTransaction as it's not related to it
2) Let's just inline this variable for better readability, no need to store it separately in this case